static_inventories/docs_library.yml

```diff
@@ -0,0 +1,15 @@
+---
+all:
+  children:
+    openstack_api:
+      hosts:
+        localhost:
+    docs:
+      hosts:
+        docs:
+          cloud_flavor: 4c4m60d
+docs_group:
```

Comment:
```suggestion
docs_library:
```

Reply:
The name of this group must match stack_name and the name of the stack's folder in `group_vars/`. It was listed wrong as `docs_group` in the `*.ini` inventory: I probably forgot to rename it there in e previous PR.